codalab/lib/download_manager.py

```diff
@@ -321,7 +328,7 @@ def _get_target_path(self, target):
         bundle_link_url = self._bundle_model.get_bundle_metadata(
             [target.bundle_uuid], "link_url"
         ).get(target.bundle_uuid)
-        if bundle_link_url:
+        if bundle_link_url and not bundle_link_url.startswith("azfs://"):
```

Comment:
Would be to refactor the check for `azfs://` in one place.  Also, let's think through what this will look like once we have `s3://`.  Probably makes sense to have a broader function that captures all these cases.